sandy/core/cov.py

```diff
@@ -2174,7 +2174,7 @@ def segmented_pivot_table(data_stack, index, columns, values, rows=10000000):
        2	0	0	1
     """
     size = data_stack.shape[0]
-    pivot_matrix = pd.DataFrame()
+    pivot_matrix = []

```

Comment:
I like this more! but to prove that it works could you make another test with `rows=1` in `sandy.cov.segmented_pivot_table(S, index=['dim1'], columns=['dim2'], values='cov')`